integration-examples/System Center Service Manager/T-SQL Queries/CMDB/Changes.sql

```diff
@@ -0,0 +1,17 @@
+-- Select open Change requests for a CI created in the last 7 days
+SELECT 
+    change.Id_9A505725_E2F2_447F_271B_9B9F4F0D190C  AS ID,
+    change.Title_9691DD10_7211_C835_E3E7_6B38AF8B8104 AS Title,
+    changeStatus.DisplayName AS Status,
+    CONCAT(CONVERT(VARCHAR(24), change.CreatedDate_6258638D_B885_AB3C_E316_D00782B8F688, 113), ' UTC') As CreatedDate,
```

Comment:
These are only samples - 113 produces a nice readable format, whereas 126 (ISO8601) does not unless the results get sent into core-timeago, which isn't appropriate in all cases.

People will need to adjust these queries in their environment to take into account things like times, since they will likely want them in their zone.